### DISCUSSION TITLE
nextflow initial installation

### DISCUSSION THREAD

**Discussion by chenyanniii** [nextflow-io/nextflow#2290] [Q&A]
Hello,

I used `conda install nextflow`to install nextflow, and not able to run nextflow. Could you help find the solution. Thank you!

When I run `nextflow run hello`

> Java HotSpot(TM) 64-Bit Server VM warning: Option --illegal-access is deprecated and will be removed in a future release.
Error: Could not find or load main class nextflow.cli.Launcher
Caused by: java.lang.ClassNotFoundException: nextflow.cli.Launcher

Other information about my system
> % which java
/usr/local/anaconda3/bin/java
% java -version
openjdk version "11.0.9.1" 2020-11-04 LTS
OpenJDK Runtime Environment Zulu11.43+55-CA (build 11.0.9.1+1-LTS)
OpenJDK 64-Bit Server VM Zulu11.43+55-CA (build 11.0.9.1+1-LTS, mixed mode)

>nano ~/.zshrc 
export PATH="/usr/local/anaconda3/bin:$PATH"
export JAVA_HOME='/Library/Java/JavaVirtualMachines/jdk-16.0.2.jdk/Contents/Home'

>% /usr/libexec/java_home -V
Matching Java Virtual Machines (2):
    16.0.2, x86_64:	"Java SE 16.0.2"	/Library/Java/JavaVirtualMachines/jdk-16.0.2.jdk/Contents/Home
    16, x86_64:	"Java SE 16"	/Library/Java/JavaVirtualMachines/jdk-16.jdk/Contents/Home
/Library/Java/JavaVirtualMachines/jdk-16.0.2.jdk/Contents/Home

**Comment by abhi18av** [comment]
Hi @chenyanniii ,

It's quite strange that your `JAVA_HOME` is not indicating the `java` version installed via `conda` located at `/usr/local/anaconda3/bin/java` 

Please update the `JAVA_HOME` in your `.zshrc` and try again.
